component-interaction/src/app/components/user-details.component/user-details.component.css

```
@@ -0,0 +1,3 @@
+.css{
```

```suggestion
.your-class-name {
```
change to a proper name